scripts/check_config.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+REPOPATH=$PWD
+exporterFile=$REPOPATH'/plugins/transformerExporter.go'
+exportedTransformers=( $( sed -n '/vdb-mcd/p' $exporterFile | cut -d' ' -f 1 | sort ) )
+
+configFile=$REPOPATH'/environments/docker.toml'
+configTransformers=( $(sed -n "/exporter\..*/p" $configFile | awk -F "." '{print $2}' | tr -d "]" | sort ) )
+
+if [ ${#exportedTransformers[@]} != ${#configTransformers[@]} ]; then
+  echo transformerExporter and config contain different transformer totals
+  exit 1
+fi
+
+for ((i=0; i<${#exportedTransformers[@]}; i++)); do
+  if [ ${exportedTransformers[$i]} != ${configTransformers[$i]} ]; then
```

Comment:
Does this depend on the order of the transformers being the same between `environments/docker.toml` and `plugins/transformerExporter.go`?

Reply:
It does. I sort both collections when I create them. I guess the O(n**2) solution wouldn't be terrible since we don't expect either collection to be large and then we wouldn't have to worry about order.

Reply:
👍 🎉